src/editTask.test.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @jest-environment jsdom
+ */
+import Task from './tasks.js';
+import displayTask from './displayTask.js';
+
+const taskObj = new Task();
+
+describe('Should test for status and content updates', () => {
+  const updateText = (id, statusCheck) => {
+    taskObj.tasks.completed = statusCheck;
+    localStorage.setItem('tasks', JSON.stringify(taskObj.tasks));
+  };
+
+  const editText = (textValue) => {
+    taskObj.tasks.description = textValue;
+    localStorage.setItem('tasks', JSON.stringify(taskObj.tasks));
+  };
+
+  document.body.innerHTML = ` <div class="edit-task" data-key="1"> <ul class="ul-edit" >
+  <li class="tick" ><input class="text-input" id="1" type="checkbox"></li>
+  <li class"text-value" ></li>
+  <li class="remove-btn" id="1"><button >&#8942;</button></li>
+</ul>
+<i class="fa-solid fa-trash-can delete-icon"></i>
+<hr>
+</div>`;
+
+  const checkBoxValue = document.querySelector('.text-input');
+  const textValue = document.querySelector('.text-value');
+
+  test('Should edit task description', () => {
+    taskObj.addNewTask(textValue);
+    document.body.innerHTML = '  <div class="todo-list"></div> ';
+    displayTask();
+    expect(taskObj.tasks).toHaveLength(1);
+  });
```

Comment:
- [ ] Let's add test for  "Clear all completed" function.

Reply:
@uwadonat okay we will add the steps to run the tests on the readme file.

Reply:
![Screenshot (65)](https://user-images.githubusercontent.com/64264883/167110404-84ac5d11-1f87-42a6-b98a-0ae0368d0e41.png)

We have included the steps and the test results on the readme file
thanks for your response we have worked on implementing the change